scripts/post-merge

```diff
@@ -26,6 +26,8 @@ else
         composer install --no-dev --prefer-dist
     fi
     if [ -f ".env" ]; then
+        echo "php artisan optimize --clever --dont-confirm=assume-no"
```

Comment:
Same question: will this command be run if updating via GUI? If so, what happens with the prompt then?

Reply:
See https://github.com/LycheeOrg/Lychee/pull/1516/files#r982539157.